tests/test_transforms/test_decomposition/test_change_points_trend_transform.py

```diff
@@ -166,15 +166,17 @@ def test_fit_transform_with_nans_in_tails(multitrend_df_with_nans_in_tails):
     transform = ChangePointsTrendTransform(
         in_column="target", change_point_model=Binseg(), detrend_model=LinearRegression(), n_bkps=5
     )
-    transformed = transform.fit_transform(df=multitrend_df_with_nans_in_tails)
-    for segment in transformed.columns.get_level_values("segment").unique():
-        segment_slice = transformed.loc[pd.IndexSlice[:], pd.IndexSlice[segment, :]][segment]
+    ts = TSDataset(multitrend_df_with_nans_in_tails, freq="D")
+    transformed = transform.fit_transform(ts=ts)
```

Comment:
Transforms works inplace now, so not to confuse someone it is better to write
```
transform.fit_transform(ts=ts)
```
without left side